### ISSUE TITLE
[Feedback] Sorting

### ISSUE THREAD

**Issue by paintedbicycle** [mathieudutour/TimeLines#3]
Might be nice to sort by time
---
Time Lines 1.0.1 (27) - me.dutour.mathieu.TimeLine
macOS 13.4.1
D421AP

**Comment by mathieudutour** [comment]
you can already reorder the contacts manually, you are suggesting to show a button to automatically sort then by timezone?

**Comment by paintedbicycle** [comment]
Yeah, with long lists it might be nice to auto sort by name or by their time zone.
